circuit-lib/circuit-lib.js/src/elgamal-babyjubjub/pointEncoding.ts

```diff
@@ -3,16 +3,6 @@ import { ExtPointType } from "@noble/curves/abstract/edwards";
 
 export type LookupTable = { [key: string]: string };
 
-/**
- * @param plaintext A 32-bit bigint
- * @returns A point on the Baby Jubjub curve
- */
-export function encode(plaintext: bigint): ExtPointType {
-  if (plaintext >= BigInt(2 ** 32)) {
-    throw new Error("The plaintext should nit be bigger than a 32-bit bigint");
-  } else return babyjubjubExt.BASE.multiplyUnsafe(plaintext);
-}
-
```

Comment:
what's the reasoning for moving encode to elgamal.ts?

Reply:
Circular dependencies. 

I'm looking at this with fresh eyes and thinking that this is probably not the best (semantically) point to move this thing, I'll do one more tiny refactoring here.